lib/iceberg/meta/validate_params.js

```diff
@@ -24,5 +25,11 @@ module.exports = (args = {}) => {
     return 'Amount & slice amount must have same sign'
   }
 
+  if (_futures) {
+    if (!_isFinite(lev)) return 'Invalid leverage'
+    if (lev < 1) return 'Leverage less than 1'
+    if (lev > 100) return 'Leverage greater than 100'
```

Comment:
Might want to make this more customization since BFX may want to offer more than 100 in the future. Also Binance offer 125x lev (which is absolutely insane btw...)